collector/tx_processor.go

```diff
@@ -270,6 +272,14 @@ func (p *TxProcessor) processTx(txIn common.TxIn) {
 		}
 	}
 
+	// Add transaction to Clickhouse
+	if p.clickhouseConn != nil {
+		err = p.clickhouseConn.AddTransaction(txIn) // send to Clickhouse
```

Comment:
Should we also write trash to a separate clickhouse table?

Reply:
good q, unsure if worth it 🤔

Reply:
let's do it as separate pr, and ask data team first